### PR TITLE
Add blog post about Scala Days 2019

### PR DESCRIPTION
**Pull request by darjutak**

### CODE REVIEW

File: blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+*The Community dinner and guided tours are included in the price of the Conference ticket, entry fee applies if you chose to join only one of those.*
+
+
+## Call for action
+
+### Submit a proposal until 20th of January
+
+We encourage you to [submit one or more proposals](https://cfp.scaladays.org/events/scaladays2019), even if you never gave a talk before. Here are some arguments:
+
+- We ask for about 600 words for an abstract for now, so your talk doesn’t need to be ready - just your idea! Share it with us. If your proposal gets accepted you will still have 4 months to develop a presentation, maybe even try it out in a local Meetup.
```
Comment (zaharidichev) [inline blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md]: I beleive it is 600 chars

Comment (darjutak) [reply blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md]: very true, great catch!!!

File: blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Speaker support
+To be eligible for speaker support, you first need to get your proposal accepted. If your company or university are not supporting you, please just reply to your “Accepted” email listing the reasons you would need a support and what kind. Our team will be communicating with you until all elements for the decision are in place and give you an answer in due time.
+
+### Become a Sponsor
+Please send us a [direct email](mailto:info@scaladays.org) to initiate this conversation. This year we are also encouraging Scala Startups to come forward as we have reserved a hype space to showcase brave, young Scala companies.
```
[the remark this answers is not in the document]
Comment (darjutak) [reply blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md]: Thank you for the comment, I had no idea about the connotations - I changed it to "special"

File: blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### More
+We would like to showcase some of the history. Please share your “throwbacks” and tag us @ScalaDays, we would like to hear from you and include some of your stories and photos in overall promotion.
```
[the remark this answers is not in the document]
Comment (darjutak) [reply blog/_posts/2019-01-17-scala-days-2019-celebrating-collaborative-success.md]: done, thanks!

**Pull request merged**